src/main/kotlin/com/terraformation/backend/species/api/SpeciesController.kt

```diff
@@ -143,7 +143,7 @@ class SpeciesController(
       organizationId: OrganizationId,
```

Comment:
Ok to remove this org id from before? Looks unused.

Reply:
Yeah, that seems fine to remove.